changelogs/fragments/comment_fail.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Since an exception was previously raised, I'd say this is more a feature than a bug fix.

```suggestion
minor_changes:
```